instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -183,15 +212,28 @@ def get_or_create_headers():
                 result = getattr(exc, "response", None)
             finally:
                 elapsed_time = max(
-                    round((default_timer() - start_time) * 1000), 0
+                    round((default_timer() - start_time)), 0
```

Comment:
nit: not sure why old duration needed to be rounded, but the new one (in seconds) should not be for sure.

Reply:
Got it. This will require some refactoring of the way I made the duration histogram array

Reply:
Fixed.